test/unit/alignment/matrix/edit_distance_matrix/edit_distance_trace_matrix.hpp

```diff
@@ -10,9 +10,6 @@
 
 #include "../../pairwise/fixture/alignment_fixture.hpp"
 
-using namespace seqan3;
-using namespace seqan3::test::alignment::fixture;
```

Comment:
Just a quick question:
Why do you delete "using namespace seqan3::test::alignment::fixture;" in this file and add it in the next file ?

Reply:
Yes, you're right. I will undo this change.